pegasus/sites.v3/hourofcode.com/views/school_not_found.haml

```diff
@@ -0,0 +1,23 @@
+#schoolnotfound
+  %label= hoc_s(:school_not_found_heading)
```

Comment:
From the screenshot, it seems it might be good if we stylistically differentiate the error message from the input labels directly below.